src/app-layout/notifications/styles.scss

```diff
@@ -7,6 +7,6 @@
 
 .notifications-sticky {
   top: 0;
-  z-index: 825;
+  z-index: 900;
```

Comment:
I looked into all existing z-indices in the repo as far as I can tell. 900 sets the notifications above the split panel (which has 851) but still below the navigation elements (1000 and beyond).

Reply:
Reverted these z-index changes because they break the (very subtle) shadows casted by the side nav and toolbars on the flashbar in Classic. We'll handle later separately how to show the flashbar on top of the split panel, which is not something specific of the stacking feature.